draft-ietf-tls-ticketrequests-02.md

```diff
@@ -0,0 +1,165 @@
+---
```

Comment:
This file is not necessary, why not have a git tag when submitting?

Reply:
I think this was just a mistake

Reply:
Yep, fat fingered.